requirements.txt

```diff
@@ -0,0 +1,2 @@
+psycopg2-binary==2.8.6
+snowflake-connector-python==2.2.8
```

Comment:
One thing that maybe you missed is that I set up the package management using `poetry`. This uses the new `.toml` files to manage packages and also it resovles dependencies a lot better than plain pip.

You can see some info here: https://github.com/bitpicky/dbt-sugar/blob/main/CONTRIBUTING.md#development

To add requirements you should then do:
```bash
poetry add <package_name> 
```

You can add `-D` if it's a developmnent requirement (which it isn't in this case)

And you should install the package via `poetry install` it will set up a venv, install the dependencies and install `dbt-sugar` in editable mode in the venv. If you want to activate it you can do `poetry shell` from the repo root and it will activate it for you. It's sooooo good!